src/features/activities/activity-form/ActivityEditForm.tsx

```diff
@@ -225,14 +233,24 @@ const ActivityEditForm = ({
             isNow={isNow}
             onTimeChange={setTime}
           />
-          {!!alert && (
-            <Alert
-              isScrollIntoView
-              severity={alert.severity}
-              title={alert.title}
-              description={alert.description}
-            />
-          )}
+        </div>
+        <div className="alerts-container">
+          {alerts?.map((alert: { id: any; severity: any; icon: string | number | boolean | React.ReactElement<any, string | React.JSXElementConstructor<any>> | React.ReactFragment | React.ReactPortal | null | undefined; title: string | number | boolean | React.ReactElement<any, string | React.JSXElementConstructor<any>> | React.ReactFragment | React.ReactPortal | null | undefined; content: string | number | boolean | React.ReactElement<any, string | React.JSXElementConstructor<any>> | React.ReactFragment | React.ReactPortal | null | undefined; }, index: number) => (
+            <div
+              key={alert.id || index}
+              className={`alert-card alert-${alert.severity}`}
+              style={{ bottom: `${20 + index * 10}px` }}
+              onClick={() => removeAlert(alert.id || index)}
+            >
+              <div className="alert-content-container">
+                <div className="alert-icon">{alert.icon}</div>
+                <div className="alert-header">
+                  <div className="alert-title">{alert.title}</div>
+                  <p className="alert-content">{alert.content}</p>
+                </div>
+              </div>
+            </div>
+          ))}
```

Comment:
I see no reason why you'd want to loop through the alerts directly here if you already have your context.
Please use the context instead.